### PR TITLE
Return ColorRGBA directly from GetColor

### PR DESCRIPTION
**Pull request by ojura**
@wohe here's that small change you mentioned in #375, make `GetColor` return `std_msgs::ColorRGBA` directly.

BTW I have two ways of making this change. In the first version (this pull request), I kept the `ColorRgb` struct for use in `HsvToRgb` because the initializer list (`{r, g, b}`) syntax makes our life easier (`ColorRGBA` can't be constructed directly like this). Another option is in https://github.com/googlecartographer/cartographer_ros/compare/master...larics:color-rgba-2, where I wrote a factory function for `ColorRGBA` which is then used in `HsvToRgb`, and this still looks relatively okay. Let me know which one you prefer. (Or if you have something completely different in mind :) )

### CODE REVIEW

**Comment by wohe** [comment]
Seems not a huge difference, but I'd say the alternative with the factory function is slightly more readable.

**Comment by ojura** [comment]
Sure. I've updated the pull request to use that version.

File: cartographer_ros/cartographer_ros/color.h [hunk 1]
```diff
@@ -17,18 +17,13 @@
...
-struct ColorRgb {
-  // r, g, b are from [0,1]
-  float r;
-  float g;
-  float b;
-};
+namespace cartographer_ros {
 
 // A function for on-demand generation of a colour palette, with every two
 // direct successors having large contrast. Initial hue is from [0, 1>.
```
Comment (wohe) [inline cartographer_ros/cartographer_ros/color.h]: Remove the sentence about the initial hue?

File: cartographer_ros/cartographer_ros/color.h [hunk 2]
```diff
@@ -17,18 +17,13 @@
...
 
-struct ColorRgb {
-  // r, g, b are from [0,1]
-  float r;
-  float g;
-  float b;
-};
+namespace cartographer_ros {
 
 // A function for on-demand generation of a colour palette, with every two
```
Comment (wohe) [inline cartographer_ros/cartographer_ros/color.h]: Consistency: US English, so `colour` -> `color`.

**Pull request merged**